src/systems/user_commands/UserCommands.cc

```diff
@@ -1362,79 +1373,142 @@ LightCommand::LightCommand(msgs::Light *_msg,
 //////////////////////////////////////////////////
 bool LightCommand::Execute()
 {
-  auto lightMsg = dynamic_cast<const msgs::Light *>(this->msg);
+  auto lightMsg = dynamic_cast<msgs::Light *>(this->msg);
```

Comment:
Is there a reason to remove the const?  You shouldn't be mutating the message, correct?

Reply:
Have to set the changes to the entity ID for each match found in the for loop.

Reply:
https://github.com/gazebosim/gz-sim/blob/149f9a28b9b8e026a90f8e65f43a53331b5fe397/src/systems/user_commands/UserCommands.cc#L1396